test/cpp/test_aten_xla_tensor_1.cpp

```diff
@@ -1400,6 +1400,79 @@ TEST_F(AtenXlaTensorTest, TestDropoutInPlace) {
   ExpectCounterChanged("xla::bernoulli", cpp_test::GetIgnoredCounters());
 }
 
+TEST_F(AtenXlaTensorTest, TestNativeDropout) {
+  torch::Tensor a = torch::rand({17, 21}, torch::TensorOptions(torch::kFloat));
+  float allowance = 0.04;
```

Comment:
Very cool! 

quick question, how is value 0.04 determined?

Reply:
Thanks, this value was referred from the existing dropout test code [here](https://github.com/pytorch/xla/blob/498be74cb14d01ec2a535d212615907f68e41a1b/test/cpp/test_aten_xla_tensor_1.cpp#L1367C1-L1383C2). They didn't mention the value 0.04 directly, but based on the [0.86, 0.94] range, the value `0.04` is the allowance. To be honest, I don't know why they choose this number.